### PR TITLE
Update the Polish translation.

### PR DESCRIPTION
**Pull request by krokots**
Fixed duplicate polish word for "lots" and "swarm".

### CODE REVIEW

**Comment by Arthusppp** [comment]
I will. Any deadline?

**Comment by Arthusppp** [comment]
Hello @krokots ! What is the idea behind your changes? 

I have compered it on this map:
[Map_0003.zip](https://github.com/ihhub/fheroes2/files/14014543/Map_0003.zip)

As far as I can see, all translations (except to one) in the fh2 are the same as in the original game (I have GoG version). 
The one exception is "swarm" - currently is "mnóstwo" and the correct translation is "mrowie".


OG/fh2
1-4 few -> mało/mało
5-9 several -> parę/parę
10-19 pack -> grupa/grupa
20-49 lots -> mnóstwo/mnóstwo
50-99 horde -> horda/horda
100-249 throng -> tłum/tłum
250-499 swarm -> mrowie/mnóstwo ?????? -> mrowie!
500-999 zounds -> setki/setki
1000-4000 legion -> legion/legion

**Comment by krokots** [comment]
There was a bug where both "lots" and "swarm" was translated as "mnóstwo". I changed "lots" to "dużo" and kept "swarm" as "mnóstwo". I don't really care how they're translated, I just needed the duplicate to be fixed.

**Comment by Arthusppp** [comment]
> There was a bug where both "lots" and "swarm" was translated as "mnóstwo". I changed "lots" to "dużo" and kept "swarm" as "mnóstwo". I don't really care how they're translated, I just needed the duplicate to be fixed.
> 
> Also personal preference : "mrowie" sounds archaic to me.

Given the above, I propose to change wrong transladion of "swarm" from "mnóstwo" to "mrowie". 

I somehow understand your second point: "mrowie" seems to be archaic word. However, the OG translates "swarm" in this way, and "mrowie" tells me more compared to "dużo". It's beacuse "dużo" is new word here, and "dużo" has not been used anywhere in this context.

File: files/lang/pl.po [hunk 1]
```diff
@@ -278,7 +278,7 @@ msgid ""
 "Several\n"
 "%{monster}"
 msgstr ""
-"Parę\n"
+"Kilka\n"
```
Comment (Arthusppp) [inline files/lang/pl.po:281]: As per my comment, I propose to revert this change.

File: files/lang/pl.po [hunk 2]
```diff
@@ -292,7 +292,7 @@ msgid ""
 "Lots of\n"
 "%{monster}"
 msgstr ""
-"Mnóstwo\n"
+"Dużo\n"
```
Comment (Arthusppp) [inline files/lang/pl.po:295]: As per my comment, I propose to revert this change.

File: files/lang/pl.po [hunk 3]
```diff
@@ -334,13 +334,13 @@ msgid "army|Few"
 msgstr "Mało"
 
 msgid "army|Several"
-msgstr "Parę"
+msgstr "Kilka"
```
Comment (Arthusppp) [inline files/lang/pl.po:337]: As per my comment, I propose to revert this change.

File: files/lang/pl.po [hunk 4]
```diff
@@ -334,13 +334,13 @@ msgid "army|Few"
...
 
 msgid "army|Pack"
 msgstr "Grupa"
 
 msgid "army|Lots"
-msgstr "Mnóstwo"
+msgstr "Dużo"
```
Comment (Arthusppp) [inline files/lang/pl.po:343]: As per my comment, I propose to revert this change.

File: files/lang/pl.po [hunk 5]
```diff
@@ -349,7 +349,7 @@ msgid "army|Throng"
 msgstr "Tłum"
 
 msgid "army|Swarm"
-msgstr "Mrowie"
+msgstr "Mnóstwo"
```
Comment (Arthusppp) [inline files/lang/pl.po:352]: As per my comment, I propose to revert this change.

File: files/lang/pl.po
Comment (Arthusppp) [inline files/lang/pl.po:1]: Please update line 316:
from 
```
msgid ""
"A swarm of\n"
"%{monster}"
msgstr ""
"Mnóstwo\n"
"%{monster}"
```
to
```
msgid ""
"A swarm of\n"
"%{monster}"
msgstr ""
"Mrowie\n"
"%{monster}"
```

**Review by Arthusppp — CHANGES_REQUESTED** [review]

**Pull request closed without merging**